ironic/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN crudini --set /etc/ironic/ironic.conf pxe ipxe_enabled true
 RUN crudini --set /etc/ironic/ironic.conf pxe pxe_config_template \$pybasedir/drivers/modules/ipxe_config.template
 
 RUN mkdir /tftpboot
+RUN echo '172.22.0.1 ostest-api.test.metalkube.org' >> /etc/hosts
```

Comment:
https://github.com/metalkube/dev-scripts/pull/29 would make this unnecessary

Reply:
Great because it the part of this patch that doesn't seem to work, when the containers is started /etc/hosts is rewritten ;-(

Reply:
I've removed this line, PR #29 solves the problem